src/esbmc/bmc.cpp

```diff
@@ -713,6 +713,7 @@ smt_convt::resultt bmct::multi_property_check(
   std::mutex result_mutex;
   // For coverage info
   int tracked_instrument = 0;
+  std::unordered_set<std::string> reached_claims;
```

Comment:
Two questions:
- It seems `tracked_instrument` is the same as `reached_claims.size()`. Do we need both?
- Is a claim indeed uniquely identified by its `claim_msg`, which seems to be a somewhat static comment? If so, could you explain how that works?

Reply:
Deleted  ‘tracked_instrument’